src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Public.Data/DataSetService.cs

```diff
@@ -157,9 +163,36 @@ private async Task<DataSetViewModel> MapDataSet(DataSet dataSet, CancellationTok
             SupersedingDataSetId = dataSet.SupersedingDataSetId,
             DraftVersion = draftVersion,
             LatestLiveVersion = latestLiveVersion,
+            PreviousReleaseIds = previousReleaseIds
         };
     }
 
+    private async Task<List<Guid>> GetDataSetPreviousReleaseIds(DataSet dataSet, CancellationToken cancellationToken)
+    {
+        var previousReleaseFileIds = dataSet
+            .Versions
+            .Select(version => version.ReleaseFileId)
+            .ToList();
+
+        return await GetReleaseIdsForReleaseFiles(
+            contentDbContext,
+            previousReleaseFileIds,
+            cancellationToken);
+    }
+
+    private static async Task<List<Guid>> GetReleaseIdsForReleaseFiles(
+        ContentDbContext contentDbContext,
+        List<Guid> releaseFileIds,
+        CancellationToken cancellationToken)
+    {
+        return await contentDbContext
+            .ReleaseFiles
+            .Include(releaseFile => releaseFile.ReleaseVersion)
```

Comment:
Minor one but you don't need an `Include` here.

Reply:
Gotcha - removed ta.  Must've been a hangover from previous attempts!